andela/mrm_api
# #161279901 Add option for meeting update notifications

#### What does this PR do?
- Adds the functionality for a user to opt in/out of meeting update notifications

#### How should this be manually tested?
- Run the server
- Run the `getUserNotificationSettings` query with `userId` as a parameter
- This should return the notification settings of the user selected
- Run the `updateNotification` mutation with `meetingUpdateNotification` as a parameter
- This should update the preferences for the user and return the notification settings

#### What are the relevant pivotal tracker stories?
[#161279901](https://www.pivotaltracker.com/story/show/161279901)

#### Screenshots
_Retrieve the current User Notification Settings_
<img width="1025" alt="screenshot 2018-10-25 at 15 18 51" src="https://user-images.githubusercontent.com/33119403/47499810-a23ef200-d869-11e8-92e3-817966868787.png">


_Change the Meeting Update Notifications field_
<img width="1009" alt="screenshot 2018-10-25 at 15 24 57" src="https://user-images.githubusercontent.com/33119403/47499981-2db88300-d86a-11e8-9dc4-71fd23f89b4e.png">
